src/commands/contents.js

```diff
@@ -16,7 +17,7 @@ const {
 } = require('../../lib/create-result');
 const { logText, logJson } = require('../../lib/log-result');
 
-exports.command = 'contents <filepath> [search]';
+exports.command = 'contents <filepath> [search] [repoList..]';
```

Comment:
I've only ever seen singular terms used in usage strings for command line tools.

Here's `sed`s:

```
sed [-Ealn] command [file ...]
```

```suggestion
exports.command = 'contents <filepath> [search] [repo..]';
```